ts/boot/configureStoreAndPersistor.ts

```diff
@@ -242,6 +242,25 @@ const migrations: MigrationManifest = {
       ...state,
       content: { ...content, idps: remoteUndefined }
     };
+  },
+  // Version 17
+  // we removed currentSelectedServices from entities.services
+  "17": (state: PersistedState) => {
```

Comment:
you don't need this migration anymore.
It has not effect and producer an error (`entities` here is undefined)